e2e/e2e.go

```diff
@@ -74,21 +74,41 @@ func BuildDependenciesList(ctx context.Context) ([]string, error) {
 	return depsList, nil
 }
 
+// getConfig gets config file from GitHub and parses it into DevenvConfig
+func getConfig(ctx context.Context, serviceName string, gh *github.Client, configFileName string) (*DevenvConfig, error) {
+	r, _, err := gh.Repositories.DownloadContents(ctx, "getoutreach", serviceName, configFileName, nil)
+	l := log.With().Str("service", serviceName).Str("file", configFileName).Logger()
+	defer func() {
+		if err := r.Close(); err != nil {
+			l.Warn().Msg("Unable to close GH reader")
+		}
```

Comment:
nit: Generally, we can throw this away; I know it's not the most intuitive thing in the world to ignore that...

Reply:
Fixed